libp2p/peersOnChannel.go

```diff
@@ -102,6 +103,9 @@ func (poc *peersOnChannel) refreshPeersOnAllKnownTopics(ctx context.Context) {
 		case <-time.After(poc.refreshInterval):
 		}
 
+		//TODO remove this
```

Comment:
is the TODO meant to be done in a future PR?

Reply:
rechecking all PR